### ISSUE TITLE
Missing tags?

### ISSUE THREAD

**Issue by mmd-osm** [Zaczero/osm-revert#18]
I've noticed that some of the nodes that have been reverted in https://www.openstreetmap.org/changeset/143128522 seem to be lacking their tags from previous versions. 

Example: https://www.openstreetmap.org/node/540424027/history

I would expect a website tag in version 19, which was present in version 13 or 15. Both of which I would consider as a good versions. The full history also shows that all versions have different timestamps, so the infamous Overpass issue with multiple versions at the same time shouldn't apply here.

**Comment by Zaczero** [comment]
![image](https://github.com/Zaczero/osm-revert/assets/10835147/71232a6b-0827-49c9-8b2d-3688516d0908)

https://osm.mapki.com/history/node/540424027

Please take note of the changeset ID that has been reverted (143124536). This revert tool automatically resolves conflicts. To restore the original set of tags, the changeset IDs [143124619](https://osm.org/changeset/143124619) and [143124645](https://www.openstreetmap.org/changeset/143124645) should also be reverted.

**Comment by mmd-osm** [comment]
Thank you for your quick and detailed analysis. So when I get this right, the idea is to compensate the changes which have been done between versions 15 and 16 as part of cs 143124536, and apply the result on 18?

It seems people are having a bit of a hard time to figure out, that vandals upload multiple versions of the same objects, and that some kind of transitive closure of these changes would be needed to do a full revert. I'm not sure if this is in scope of Thanos?

**Comment by Zaczero** [comment]
Yes, this is within the scope of Thanos (full revert + revert to a specific version). In many cases, individuals revert certain changesets without realizing the full extent of the issue. I'm considering adding an informational notice to the page. Do you have any suggestions for what it should say?

**Comment by mmd-osm** [comment]
DWG folks are probably in a better position to suggest a meaningful text here, since they're most impacted by partial reverts.

However, maybe you could check, if the changeset to be reverted has been created by a user with an active block, and suggest to the user  to align with the DWG first, or even don't let users perform the revert at all. DWG probably has an opinion on this topic. I would again recommend to reach out to them.
